components/services/bat_ledger/public/interfaces/bat_ledger.mojom

```diff
@@ -193,6 +193,8 @@ interface BatLedgerClient {
   SetDoubleState(string name, double value);
   [Sync]
   GetStringState(string name) => (string value);
+  [Sync]
+  SetStringStateSync(string name, string value) => ();
```

Comment:
It seems to me that all of the `Set*State` methods are going to be prone to the same race condition.

As far as I know, all the "Set" methods are used exactly as if they were sync. Instead of introducing a "Sync" version of `SetStringState`, can we just add the `[Sync]` attribute to all of these "SetState" methods?

Reply:
Done, thanks for the idea!